carball/analysis/stats/utils/panda_utils.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+
+
+def return_time_by_player(data_frame: pd.DataFrame, players_data_frame: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I think this function is better named as `sum_deltas_by_player_name`

also the second parameter (`players_data_frame`) seems to be a `pd.Series`.